src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/client/model/Builds.java

```diff
@@ -10,12 +11,18 @@
  */
 public class Builds implements JiraRequest {
     private List<JiraBuildInfo> builds;
+    private Properties properties;
 
     public Builds(final JiraBuildInfo jiraBuildInfo) {
         this.builds = Collections.singletonList(jiraBuildInfo);
+        this.properties = new Properties("jenkins/builds", jiraBuildInfo.getPipelineId());
```

Comment:
We don't need have the entity type as part of the property key, we already have this information in a different field. I suggest that we keep it simple and use the following property for both builds and deployments:

```
source = jenkins
```